auth-web/vite.config.ts

```diff
@@ -64,7 +64,12 @@ export default defineConfig({
       '@bcrs-shared-components/mixins': path.resolve(__dirname, './node_modules/@bcrs-shared-components/mixins/index.ts'),
       '@bcrs-shared-components/enums': path.resolve(__dirname, './node_modules/@bcrs-shared-components/enums/index.ts'),
       // Fix for module decorator unit tests fail
-      'vuex-module-decorators': path.resolve(__dirname, './node_modules/vuex-module-decorators/dist/esm/index.js')
+      'vuex-module-decorators': path.resolve(__dirname, './node_modules/vuex-module-decorators/dist/esm/index.js'),
+      // Fix for multiple vue detected on fas-ui issue, set alias to force fas-ui to use /vue
+      // Conditionally set the Vue alias only for non-test environments
+      ...(process.env.NODE_ENV !== 'test' && {
+        'vue': path.resolve(__dirname, './node_modules/vue/dist/vue.esm.browser.js')
```

Comment:
AccountInformation.spec.ts line 93 failed after I add -> 'vue': path.resolve(__dirname, './node_modules/vue/dist/vue.esm.browser.js')

Reply:
When you look at the page manually, does it have this side effect? 
Also alot of these resolves will go away when we upgrade to Vue3

Reply:
I updated the unit test instead